README.md

```diff
@@ -292,6 +290,50 @@ import hljs from 'highlight.js';
 import 'highlight.js/styles/github.css';
 ```
 
+### Browser ES6 Modules
+
+*Note: You need to install `@highlightjs/cdn-assets` package instead of `highlight.js`. See [Download prebuilt CDN assets](#download-prebuilt-cdn-assets)*
+
+To import the library and register only those languages that you need:
+
+```js
+import hljs from './node_modules/@highlightjs/cdn-assets/es/core.js';
```

Comment:
What is the `./`?  Is this assuming a build system or nobuild?  Don't real in-browser ES6 imports need to be full URLs?

Reply:
You need to check this: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#importing_features_into_your_script
Also check below for using bare specifier (`import hljs from '@highlightjs/cdn-assets/es/core.js';`) using importmap.
Code in `@highlightjs/cdn-assets` package is browser compatible, so nobuild.

Reply:
Hello, is this change somehow related to https://github.com/rails/importmap-rails/issues/270 ?

Reply:
I'm familiar with this actually, I was thinking of someone pulling from from a CDN I suppose...   I'll look at this and tweak the examples.  I'm not going to recommend that people use `node_modules` though...

I was also thinking of the FIRST import, which is (surely?) going to be relative to the HTML file... relative references for assets are such a pain (IMHO) on a large website.  I imagine named imports vs importmap solve a lot of that though.

Taking a look at this more now.